<launch>
  <arg name="project_dir" default="" />
  <arg name="bag_file" default="rs_camera.bag" />
  <arg name="playback_speed" default="1" />
  <arg name="start_time" default="0.0" />
  <arg name="enable_pointcloud" default="true" />

  <include file="$(find realsense_ros_tools)/launch/include/_rs_reproduce.launch">
    <arg name="enable_pointcloud" value="$(arg enable_pointcloud)" />
  </include>

  <!-- Play rosbag file -->
  <param name="/use_sim_time" value="true" />
  <node name="rs_bag_player" pkg="rosbag" type="play" output="screen"
    args="--clock -r $(arg playback_speed) -s $(arg start_time) $(env HOME)/Downloads/$(arg project_dir)/$(arg bag_file)" />

</launch>